platform/core-api/src/com/intellij/psi/CommonClassNames.java

```diff
@@ -28,6 +28,7 @@
 
   String JAVA_LANG_ITERABLE = "java.lang.Iterable";
   String JAVA_UTIL_ITERATOR = "java.util.Iterator";
+  String JAVA_UTIL_LIST_ITERATOR = "java.util.ListIterator";
```

Comment:
it's more common than navigatable collections but still not common enough for this class